modules/exploits/linux/local/ueb_priv_esc.rb

```diff
@@ -0,0 +1,175 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info, {
+      'Name'           => 'Unitrends UEB 10 Privilege Escalation',
+      'Description'    => %q{
+        It was discovered that the Unitrends Backup (UB) before 10.1.0 libbpext.so
+        authentication could be bypassed with a SQL injection, allowing a remote
+        attacker to place a privilege escalation exploit on the target system and
+        subsequently execute arbitrary commands.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Cale Smith', # @0xC413
+          'Benny Husted', # @BennyHusted
+          'Jared Arave' # @iotennui
+        ],
+      'DisclosureDate' => 'Mar 14 2018',
+      'Platform'       => 'linux',
+      'Arch'           => [ARCH_X86],
+            'References'     =>
+        [
+          ['URL', 'https://support.unitrends.com/UnitrendsBackup/s/article/000006003'],
+          ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2018-6329'],
```

Comment:
same as above

Reply:
also not needed because the CVE reference below will link up to the cve reference